docs/physics/est_and_conv/estimators.ipynb

```diff
@@ -0,0 +1,357 @@
+{
```

Comment:
<p>Remove "for example" in 3rd sentence first paragraph</p><p>Remove parentheses around definition of V</p><p>"The latter equation" in the sentence after defining Trad and W is unclear.</p><p>Can you clarify that l and u are lower and upper in the J_blue? What is b?</p><p>"real nu_bar estimator" could use the proper nu_bar symbol</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>